rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties

```diff
@@ -99,3 +99,6 @@ sample-collection-form-original.step3.vtm_brand.label = Merk VTM?
 sample-collection-form-original.step3.vtm_brand.options.lmk.text = LMK
 sample-collection-form-original.step3.doctor_phone_number.v_required.err =
 sample-collection-form-original.step2.lbl_sample_expired.text = Sampel telah kedaluwarsa!
+sample-collection-form-original.step11.title = Pengumpulan sampel
+sample-collection-form-original.step11.next_label = OK
+sample-collection-form-original.step11.lbl_high_heat_exposure.text = RDT ini telah ditandai karena paparan panas tinggi dan keandalannya telah dikompromikan. Harap gunakan RDT lain.
```

Comment:
In future, translations to other languages can be left blank. A separate process is used to add them.

Reply:
translation removed.